lib/src/models/replied_message_configuration.dart

```diff
@@ -38,6 +38,30 @@ class RepliedMessageConfiguration {
   final double? opacity;
   final ReplyMessageWithReturnWidget? repliedMessageWidgetBuilder;
 
+  /// Auto scrolls to original message when tapped on replied message.
+  /// Defaults to true.
+  final bool enableScrollToRepliedMsg;
+
+  /// Highlights the text by changing background color of chat bubble for
+  /// given duration. Default to true.
+  final bool enableHighlightRepliedMsg;
+
+  /// Chat bubble color when highlighted. Defaults to Colors.grey.
+  final Color highlightColor;
+
+  /// Chat will remain highlighted for this duration. Defaults to 500ms.
+  final Duration highlightDuration;
+
+  /// When replied message have image or only emojis. They will be scaled
+  /// for provided [highlightDuration] to highlight them. Defaults to 1.2.
+  final double highlightScale;
+
+  /// Time taken to auto scroll to original message. Default to 300ms.
+  final Duration highlightScrollDuration;
+
+  /// Animation curve for auto scroll. Defaults to Curves.easeIn.
+  final Curve highlightScrollCurve;
+
```

Comment:
Please make one model for above all properties. You may reference RepliedMessageConfiguration class